transformation_test/testdata/windows_event_log_v1/output_otel.yaml

```diff
@@ -42,3 +42,44 @@
       type: gce_instance
     timestamp: 2025-07-02T20:39:32.339006100Z
   partialSuccess: true
+- collector_errors:
+  - caller: ottl@v0.126.0/parser.go:395
+    error: "error getting value in ottl.StandardTimeGetter[github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/ottllog.TransformContext]: expected string but got nil"
```

Comment:
Why are these errors happening? They don't seem right (and this will result in dropped logs, IIRC). Either the time parsing isn't working as expected (wrong field name?) or there are cases where the `system_time` is null and we need to handle that in the convert function.